app/build.gradle

```diff
@@ -278,11 +284,8 @@ dependencies {
 
     compile "android.arch.lifecycle:extensions:$architecture_components_version"
 
-    compile ("com.google.code.findbugs:annotations:3.0.1") {
-        // We really only need the SuppressFBWarnings annotation, everything else can be ignored.
-        // Without this we get weird failures due to dependencies.
-        transitive = false
-    }
+    compileOnly 'net.jcip:jcip-annotations:1.0'
```

Comment:
This replaces the suppress annotations etc.? Is there no official package from spotbugs?

Reply:
We should depend on both of spotbugs-annotations and net.jcip:jcip-annotations:1.0 instead of findbugs:annotations. This can be found in Spotbugs documentation.